pageserver/src/walreceiver/walreceiver_connection.rs

```diff
@@ -187,6 +190,10 @@ pub async fn handle_walreceiver_connection(
             replication_message = physical_stream.next() => replication_message,
         }
     } {
+        // We use this timeline for the entire replication message processing, since
+        // `cancellation` above is also tracked on this level and unconditionally
+        // interrupting byte streams could be bad
```

Comment:
What do you mean by `tracked on this level`?

Reply:
That there's no cancellation token below. Had reworded the docs slightly.